src/main/java/ch/teachu/teachuapi/lookup/dto/LookupStudentRequest.java

```diff
@@ -0,0 +1,17 @@
+package ch.teachu.teachuapi.lookup.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class LookupStudentRequest extends LookupRequest {
+    private UUID schoolClassId;
+    private UUID notTakenExamId;
```

Comment:
What is a notToken?

Reply:
notTAKEN. You can give an examId and every student who has not taken the exam will be returned.